### PR TITLE
Add the API key variable to Lambda

### PR DESCRIPTION
**Pull request by VirginiaDooley**
Ref https://app.asana.com/0/1204880927741389/1205760666515689/f

Following the [enforcement of API keys on YNR](url), imports are sometimes broken or incomplete. This change:

- adds the YNR API variable to the Lambda for the importers to access
- adds the YNR API variable to every API call

### CODE REVIEW

File: wcivf/apps/elections/tests/test_helpers.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def test_do_import_with_params(self, importer):
         importer.do_import(params={"election_date": "2021-05-06"})
         importer.ee_helper.prewarm_cache.assert_not_called()
 
-        expected_url = f"{settings.YNR_BASE}/api/next/ballots/?election_date=2021-05-06&page_size=200"
+        expected_url = f"{settings.YNR_BASE}/api/next/ballots/?election_date=2021-05-06&page_size=200?{settings.YNR_API_KEY}"
```
Comment (GeoWill) [inline wcivf/apps/elections/tests/test_helpers.py:176]: I think this should be `auth_token={settings.YNR_API_KEY}`

File: wcivf/apps/parties/management/commands/import_parties.py [hunk 2]
```diff
@@ -6,7 +6,13 @@
 
 class Command(BaseCommand):
     def handle(self, **options):
-        next_page = settings.YNR_BASE + "/api/next/parties/?page_size=200"
+        next_page = (
+            settings.YNR_BASE
+            + "/api/next/parties/?page_size=200?"
+            + "&auth_token="
+            + settings.YNR_API_KEY
+        )
```
Comment (GeoWill) [inline wcivf/apps/parties/management/commands/import_parties.py:14]: f string?

File: wcivf/apps/elections/tests/test_helpers.py (hunk 1 above)
Comment (VirginiaDooley) [reply wcivf/apps/elections/tests/test_helpers.py:176]: is it `&auth_token..` or `?auth_token`?

File: wcivf/apps/parties/management/commands/import_parties.py (hunk 2 above)
Comment (VirginiaDooley) [reply wcivf/apps/parties/management/commands/import_parties.py:14]: It renders correctly without the f string

File: wcivf/apps/elections/import_helpers.py [hunk 3]
```diff
@@ -224,6 +231,7 @@ def __init__(
         self.force_current_metadata = force_current_metadata
         self.recently_updated = recently_updated
         self.base_url = base_url or settings.YNR_BASE
+        self.headers = default_header or settings.YNR_API_KEY
```
[the remark this answers is not in the document]
Comment (VirginiaDooley) [reply wcivf/apps/elections/import_helpers.py]: Changed with 2a933da0ce40b746476ac914d7dea0a99d7341b9

**Review by GeoWill — APPROVED** [review]

**Pull request merged**